arbitrum-docs/getting-started-users.mdx

```diff
@@ -10,32 +10,95 @@ import PublicPreviewBannerPartial from './partials/_public-preview-banner-partia
+- Chain ID: `421613`
+- Currency Symbol: `AGOR`
+- Block Explorer URL: `https://goerli.arbiscan.io`
+
+##### Arbitrum One:
+- Network Name: `Arbitrum One`
+- New RPC URL: `https://arb1.arbitrum.io/rpc`
+- Chain ID: `42161`
+- Currency Symbol: `ETH`
+- Block Explorer URL: `https://arbiscan.io`
+
+##### Arbitrum Nova:
+- Network Name: `Arbitrum Nova`
+- New RPC URL: `https://nova.arbitrum.io/rpc`
+- Chain ID: `42170`
+- Currency Symbol: `ETH`
+- Block Explorer URL: `https://nova.arbiscan.io/`
+
+If you’re a developer you can also rely on some popular third party node providers to interact with public Arbitrum chain, see the list [here](node_running/node-providers.mdx).
+
+#### 2- Deposit Your ETH/ Tokens (L1 → L2)
+
+To move your Ether and tokens to Arbitrum chains, visit [bridge.arbitrum.io](https://bridge.arbitrum.io/). 
+Login to the bridge with your wallet and make sure you are connected to the Ethereum mainnet network.
+Now choose the desired network you wish to bridge to, e.g., Arbitrum One or Arbitrum Nova. Note that Arbitrum Goerli only appears if you are connected to the Goerli network rather than the Ethereum Mainnet.
+
+![Add the desired Arbitrum Network to your MetaMask wallet](assets/getting_started_users_2.png)
+
+Select the token you want to bridge in the token drop-down menu. You can also enable/disable the token lists by clicking `Manage Token lists` button on the bottom right corner of the drop-down menu.
+
+![Select the token to bridge](assets/getting_started_users_3.png)
+
+Enter the amount of ETH/ tokens you want to bridge over in the “Mainnet” field and then press `deposit`. Follow the prompts on MetaMask.
+
+![Enter the amount of tokens to bridge](assets/getting_started_users_4.png)
+
+After you submit the transaction through MetaMask you can expect your funds to arrive on Arbitrum One within roughly 10 minutes to an hour (depending on the chain congestion).
+
+Also make sure your MetaMask wallet is set to any of the Arbitrum One **or** Arbitrum Nova or **or** Arbitrum Goerli networks (depending on which one you chose to bridge to) so you can see your funds when they arrive.
+
+#### 3- Withdraw Your ETH/ Tokens (L2 -> L1)
+
+:::info There's At Least A 7 Day Withdrawal Period for Arbitrum One and Nova Networks
+Once you move your funds from Arbitrum One or Nova through the Arbitrum bridge, you will have to wait for at least 7 days to recieve them on Ethereum Mainnet.
+
+:::
+
+To move your funds back to Ethereum Mainnet, your need to be logged in to the [Arbitrum bridge](https://bridge.arbitrum.io/) with your wallet and make sure you are connected to he Arbitrum network you want to withdraw from (Arbitrum One, Arbitrum Nova, or Arbitrum Goerli).
+Select the token you want to bridge in the token drop-down menu. 
+
+![Select the token to withdraw](assets/getting_started_users_5.png)
+
+Enter the amount of ETH/ tokens you want to bridge over in the `L2` field and then press `withdraw`. Follow the prompts on MetaMask.
```

Comment:
There isn't an actual `L2` field. Should be describe this differently?
Maybe "in the Arbitrum field" ? (Not 100% convinced)

Reply:
"in the Arbitrum field" makes sense. I think it'll be clear as we show that within the screenshot attached as well.